src/App.js

```diff
@@ -43,13 +35,13 @@ const App = () => {
     getSearchResults(query)
```

Comment:
Navigating from search movies is not good practice. It'd be great it'd be separated from this method